pallets/proofs-dealer/src/utils.rs

```diff
@@ -134,8 +134,7 @@ where
         ensure!(root == Self::default_forest_root(), Error::<T>::ZeroRoot);
 
         // Get last tick for which the submitter submitted a proof.
-        let last_tick_proven = match LastTickProviderSubmittedProofFor::<T>::get(submitter.clone())
-        {
+        let last_tick_proven = match LastTickProviderSubmittedProofFor::<T>::get(*submitter) {
```

Comment:
Not complaining, but why these changes in this PR?

Reply:
Just to make clippy happy.
When I run `cargo clippy` locally it showed this. Maybe I can remove these and put it in another PR just for small fixes?

Reply:
Personally I'm fine squeezing it in this PR!